src/api/app/models/bs_request.rb

```diff
@@ -994,6 +1004,27 @@ def required_checks
     target_project_objects.pluck(:required_checks).flatten.uniq
   end
 
+  # Collection of grouped open reviews
+  def grouped_reviews
```

Comment:
Same as above, this shouldn't be in the `BsRequest` model. We should instead use a query object. I have created a [card](https://trello.com/c/NysbjAZj/2659-refactor-how-reviewers-with-an-open-review-are-fetched) in our backlog to follow up on this.

Reply:
I've removed it during refactoring